plugins/search-backend-module-elasticsearch/src/engines/ElasticSearchSearchEngine.ts

```diff
@@ -171,9 +171,21 @@ export class ElasticSearchSearchEngine implements SearchEngine {
   }
 
   /**
-   * Create a custom search client from the derived elastic search
-   * configuration. This need not be the same client that the engine uses
-   * internally.
+   * Create a custom search client from the derived search client configuration.
```

Comment:
Is it a good idea to update the documentation here as well?
https://backstage.io/docs/features/search/search-engines#elasticsearch

Reply:
Yup!  That was the one-liner change in the other file--was there any other change needed in there?

Reply:
Maybe you want to rephrase this part of the text for clarity or update the code snippet using generics by including your new comments?
![image](https://user-images.githubusercontent.com/6290749/211546881-ab84ae16-6979-4a14-a9f2-efc801bea922.png)